--- cmd_kick.rs	2025-01-08 14:17:47.755248601 -1000
+++ cmd_kick.new	2025-01-08 14:19:28.359316139 -1000
@@ -29,21 +29,9 @@
             return Err(InvalidConsumption(Some(ARG_TARGET.into())));
         };
 
-        let target_count = targets.len();
 
-        for target in targets {
-            target
-                .kick(TextComponent::text("Kicked by an operator"))
-                .await;
-        }
-
-        let msg = if target_count == 1 {
-            TextComponent::text("Player has been kicked.")
-        } else {
-            TextComponent::text(format!("{target_count} players have been kicked."))
-        };
-
-        sender.send_message(msg.color_named(NamedColor::Blue)).await;
+        let msg = TextComponent::text("This command has been disabled on the public server... Sorry!");
+        sender.send_message(msg.color_named(NamedColor::Red)).await;
 
         Ok(())
     }
