--- cmd_kill.rs	2025-01-08 14:17:43.205213481 -1000
+++ cmd_kill.new	2025-01-08 14:26:12.668335160 -1000
@@ -28,19 +28,8 @@
             return Err(InvalidConsumption(Some(ARG_TARGET.into())));
         };
 
-        let target_count = targets.len();
-
-        for target in targets {
-            target.living_entity.kill().await;
-        }
-
-        let msg = if target_count == 1 {
-            TextComponent::text("Entity has been killed.")
-        } else {
-            TextComponent::text(format!("{target_count} entities have been killed."))
-        };
-
-        sender.send_message(msg.color_named(NamedColor::Blue)).await;
+        let msg = TextComponent::text("This command is temporarially disabled on the public server... Sorry!");
+        sender.send_message(msg.color_named(NamedColor::Red)).await;
 
         Ok(())
     }
